cypress/page/shopping-cart-page.ts

```diff
@@ -3,7 +3,7 @@ class ShoppingCartPage {
     private goToCheckoutBtn: string;
 
     constructor() {
-        this.goToCartBtn = "[style*='display: block;'] .button-container > a";
+        this.goToCartBtn = ".button-container > .button-medium";
         this.goToCheckoutBtn = ".cart_navigation span";
```

Comment:
use an attribute of the tag to indicate we are referring to the proceed to checkout button

Reply:
@mafeZapata Please take into account this comment